common/io/ExtendedSerial.cpp

```diff
@@ -32,6 +32,10 @@
 #ifdef HAVE_STROPTS_H
 // this provides ioctl() definition without conflicting with asm/termios.h
 #include <stropts.h>
+#else
+// this provides a minimal ioctl() definition for modern systems
+// which don't have <stropts.h> anymore
+#include <ola/io/IOCtl.h>
```

Comment:
I think from a bit of reading Linux shouldn't necessarily have stropts.h (as it's a POSIX thing).

Reply:
Yes, true, as I understand it, the Streams interface probably wasn't ever even really implemented, the header files were mostly present for decades before it was finally purged, I don't recall where but I read something along those lines in the past couple of days..